File: Projects/React-Project/ems-app/src/model/Department.js
export default class Department{
    depId ;
    depName;

    constructor(){
        this.depId=0;
        this.depName="";
    }

    setDepId(id){
        this.depId=id;
    }
    setDepName(name){
        this.depName=name;
    }
}

File: SpringBoot1/SB-EMS Project/SB-EMS-Project/src/main/java/com/ems/model/Department.java
package com.ems.model;

import java.util.List;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.OneToMany;
import jakarta.persistence.Table;

@Entity
@Table(name="department")
public class Department {
	
	@Id
	@Column(name="depid")
	private int did;
	
	@Column(name="depname")
	private String dname;
	
	@OneToMany(mappedBy = "department")
	private List<Employee> employees;

	public int getDid() {
		return did;
	}

	public void setDid(int did) {
		this.did = did;
	}

	public String getDname() {
		return dname;
	}

	public void setDname(String dname) {
		this.dname = dname;
	}

}
